coalib/bears/Bear.py

```diff
@@ -288,6 +357,24 @@ def run_bear_from_section(self, args, kwargs):
                 self.name), str(err))
             return
 
+        debug_bears = kwargs.get('debug_bears')
+        value_list = self.parenthesis_split(str(debug_bears))
```

Comment:
I don't think this is necessary, why can't you interpret the setting as al list right away? look here: https://github.com/coala/coala/blob/master/coalib/settings/Setting.py#L84